CafeTime/CafeTime/AuthenticatedUserMainView.swift

```diff
@@ -55,12 +55,13 @@ class AuthenticatedUserMainView: UIView {
     }
     
     private func setUpConstraints() {
+        
```

Comment:
please check all files before commit. Changes like this one should not be included